<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>123</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diablo 3 helper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>88</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="key1cb">
       <property name="text">
        <string>Key 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="key2cb">
       <property name="text">
        <string>Key 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="key3cb">
       <property name="text">
        <string>Key 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="key4cb">
       <property name="text">
        <string>Key 4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
